### ISSUE TITLE
Ctrl+O does not open a /etc folder (one of the first steps in tutorial)

### ISSUE THREAD

**Issue by paul-alkhimov** [trogdoro/xiki#168]
When I press Ctrl+O on etc line, it says "No shell action is installed. Type ^S...".
Here is the screencast: https://asciinema.org/a/2vppjk2bj9frrek45bu2iyzfy

**Comment by Strasser-Pablo** [comment]
I have the same problem.

**Comment by trogdoro** [comment]
Sorry for being so slow in responding. Do you have a ~/xiki/ls.xiki file? If so, what are its contents?

**Comment by Strasser-Pablo** [comment]
I don't have a ls.xiki file.
$ ls
bar.xiki  bookmarks  commands  disk.xiki  _error_getting_files.xiki  examples.xiki  git.xiki  links.xiki  memorize.xiki  misc  notes.xiki  search_for_notes.xiki  sessions  sessions.xiki  tutorial.xiki  xsh.xiki

**Comment by Strasser-Pablo** [comment]
However I have a ls.rb in commands.

**Comment by trogdoro** [comment]
https://gist.github.com/trogdoro/b314daef946e51682439532597488f0f

Please put this in ~/xiki/ and tell me if that helps.

**Comment by sezycei** [comment]
Same issue here. Installed on Arch Linux, using zsh. Adding that command file worked, but it's not there by default. Might consider editing the tutorial or adding that to the default set of command scripts.

**Comment by Strasser-Pablo** [comment]
Same for me. It now works. Thanks.

**Comment by trogdoro** [comment]
Awesome, thanks so much guys! Not sure why that file didn't transfer over
during the install. After the XikiHub Kickstarter gains momentum, let's
pair together and see we can figure out why.

Ps the Kickstarter just launched at -
https://www.kickstarter.com/projects/xiki/xikihub-the-social-command-line

--Craig

**Comment by Saikyun** [comment]
Installed today and had the same problem on osx using the standard terminal. Your file fixed it though, so thanks for that! Sadly google hadn't indexed this yet, so it was a bit hard to find this fix. :p

**Comment by thams** [comment]
shouldn't this be committed back on the xiki master branch?

**Comment by trogdoro** [comment]
This problem is only happening for some people. So far no one has
identified a code fix, for the times when the problem does occur. If you've
got one, it would be super awesome if you would contribute it!

--Craig

**Comment by thams** [comment]
so here's something strange... 

I'm pretty sure this worked originally for me. So, possibly this file got removed after the fact. Is that a clue?

**Comment by thams** [comment]
Would it be worth adding something that checks for the file presence afterwards and barks if it isn't there? That might reveal whether the file is not getting copied vs. getting removed

**Comment by skjorrface** [comment]
I just tried to add ls.xiki to ~/xiki/, the only thing that changes is that <C-o> will try to edit a file with the same name of the directory like this:

![schermata 2017-10-18 13 56 59](https://user-images.githubusercontent.com/9011806/31717463-4110d656-b40c-11e7-9ca8-886720aede0f.png)

If I type "ls /" outside xsh and then <C-o> it correctly opens / though, the strange behaviour happens only inside xsh. I'm using zsh and tmux + Ubuntu 17.10. Any hint?

**Comment by scottdky** [comment]
I am having the same problem as skjorrface. I created the ls.xiki file as instructed, and when trying to open the /etc dir as in the tutorial. I get the same new file being edited.

**Comment by dreadjesus** [comment]
Running Ubuntu 17.10 with bash. Having the same problem, and when i use the file your linking it simply tries to edit it. 
and if i use "xsh ls/" its show alright but i cant open folders or retrieve the cd part when i quit.

**Comment by Shadoukun** [comment]
Also having this problem on Arch.

I also use zsh + tmux.

**Comment by jmwatte** [comment]
Also having this problem on Antergos. 
No shell action installed.
put the ls.xiki file in /.xiki
still...
no shell action installed.

**Comment by ChaseCrum** [comment]
I read a tweet from the Linux Foundation about this program, and I have been glued to it ever since. There's no native package for it in SUSE repos, but I think I can fix that <hehehe>. I'm having the same problem these guys are having, and once I inserted the file here, ls didn't work anymore. So I'm trying to get an editor to open files with, and that isn't doing it.
-Chase

**Comment by mbusch-regis** [comment]
Followed the directions, put ls.xiki in my ~/xiki directory. now doing an ls <ctrl-O> simply opens and prints the ls.xiki file...

**Comment by wjvander** [comment]
Yup, same problem. Fresh default install on linux mint, add ls.xiki file as instructed, still getting the same "no action" message

**Comment by mwesterhof** [comment]
confirmed on ubuntu 16.04 as well. after adding the file, it still doesn't work

**Comment by DPortillo821** [comment]
Adding that file worked for me. Latest version of high sierra installed (10.13.5)
